tests/projects/SelfContained_Trimming_Test/check.ps1

```diff
@@ -14,7 +14,7 @@ if (-not ($output -eq $expected))
 }
 
 # Checking that FSharp.Core binary is of expected size (needs adjustments if test is updated).
-$expected_len = 246272  # In bytes
+$expected_len = 248320 # In bytes
```

Comment:
OMG ... this is really a test?

Reply:
Yes, this is a CI test.
And is hell of annoying if we have several affecting PRs in the queue, because I need to be updating this number on each merge of main :((

Reply:
> OMG ... this is really a test?

This is by design, I've talked about it multiple times.
Here we pretty much ensure we don't change codegen changes accidentally, and also we want to make sure that when we update SDK - it doesn't bring unwanted changes to trimming.